#ver=0.1.0
---
topic_GPS:
    topicName : "gnss" # The _id number will added in suffix, i.e. "<topicName>_<id>"
    publishInterval_s : 0.1 # Maximum: 10Hz

GPS_prop:
    gpsDevice : "/dev/ttyUSB0"
    gpsBaud_dec : 115200
    ntripCaster : "61.220.23.236" # "rtk2go.com"
    ntripPort : "2101"
    ntripMountpoint : "/MIRDC01" # For rtk2go: "/BX4ACP", "/Navi-test", "/geec01", "/taitung_agri"
    ntripUsername : "123@gmail.com" # Not necessary
    ntripPassword : "none" # Not necessary

procedure_timeout_ms_prop:
    gpsModule : 6000.0
    msgUpdate : 6000.0
    msgPublish : 200.0

# Settings for Params class under vehicle_interfaces/params.h
generic_prop:
    # Node name and namespace. The namespace will automatically adding prefix at node name, topic name, service name, etc..
    namespace : "V0"
    nodeName : "gnss"

    # Device ID, the ID will automatically formed the name of nodes and publisher topics. For instance, 
    # the node name will be formed as "<nodeName>_<id>_node", and the publisher topic name will be formed as "<topic_name>_<id>". 
    # Note: If the topic use to be subscriptions, then the topic name will not be formed.
    id : 0
